app/util/analytics/application_info.py

```diff
@@ -83,7 +86,7 @@ def version(self):
 
     @property
     def nodes_count(self):
-        return self.client.get_confluence_nodes_count()
+        return len(self.client.get_confluence_nodes_count())
```

Comment:
The name of get_confluence_nodes_count() looks confusing. It should rerun nodes, but not count.